app/controllers/orders/index.js

```diff
@@ -4,5 +4,17 @@ export default searchModule.extend({
 
   searchModelName: "designation",
   unloadAll: true,
-  minSearchTextLength: 2
+  minSearchTextLength: 2,
+
+  onItemLoaded(record) {
+    const orgId = record.get("gcOrganisationId");
+    if (orgId) {
+      this.store.findRecord("gc_organisation", orgId, { reload: false });
+    }
+    if(record.constructor.toString() === "stock@model:designation:") {
```

Comment:
@Darshan-Chauhan Would you know what this condition is about ? (was previously in search_module.js)

Reply:
Yeah, I think when searching orders, we were having trouble loading OrdersPackages associated with Order. So if returned record is designation model then we're requesting ordersPackages associated with that designation object.

But I think we're getting ordersPackages data with Orders now so we don't need to fire extra request for ordersPackages.

@patrixr , you can check if we're getting OrdersPackages in Order search response then we can remove this.